.aws/src/SqsLambda.ts

```diff
@@ -0,0 +1,76 @@
+import { Resource } from 'cdktf';
+import { Construct } from 'constructs';
+import { config } from './config';
+import { PocketVPC } from '@pocket-tools/terraform-modules';
+import { PocketSQSWithLambdaTarget } from '@pocket-tools/terraform-modules';
+import { LAMBDA_RUNTIMES } from '@pocket-tools/terraform-modules';
+import { ssm } from '@cdktf/provider-aws';
+import { PocketPagerDuty } from '@pocket-tools/terraform-modules';
+
+export class SqsLambda extends Resource {
+  constructor(
+    scope: Construct,
+    private name: string,
+    vpc: PocketVPC,
+    pagerDuty?: PocketPagerDuty
+  ) {
+    super(scope, name);
+
+    const { sentryDsn, gitSha } = this.getEnvVariableValues();
+
+    new PocketSQSWithLambdaTarget(this, 'sqs-event-consumer', {
+      name: `${config.prefix}-Sqs-Event-Consumer`,
+      batchSize: 10,
+      batchWindow: 60,
+      sqsQueue: {
+        maxReceiveCount: 3,
+        visibilityTimeoutSeconds: 300,
```

Comment:
nit pick: should we lower this down to `150`? Asking because that's what we have for the curation migration datasync lambda

Reply:
hmm, this was the default we (also) copied from elsewhere. I'll leave it for now and revisit it if we need to 👍